src/main/java/com/puppycrawl/tools/checkstyle/ConfigurationLoader.java

```diff
@@ -492,7 +471,15 @@ private final class InternalLoader
          */
         private InternalLoader()
                 throws SAXException, ParserConfigurationException {
-            super(createIdToResourceNameMap());
+            super(Map.of(
+                DTD_PUBLIC_ID_1_0, DTD_CONFIGURATION_NAME_1_0,
+                DTD_PUBLIC_ID_1_1, DTD_CONFIGURATION_NAME_1_1,
+                DTD_PUBLIC_ID_1_2, DTD_CONFIGURATION_NAME_1_2,
+                DTD_PUBLIC_ID_1_3, DTD_CONFIGURATION_NAME_1_3,
+                DTD_PUBLIC_CS_ID_1_0, DTD_CONFIGURATION_NAME_1_0,
+                DTD_PUBLIC_CS_ID_1_1, DTD_CONFIGURATION_NAME_1_1,
+                DTD_PUBLIC_CS_ID_1_2, DTD_CONFIGURATION_NAME_1_2,
+                DTD_PUBLIC_CS_ID_1_3, DTD_CONFIGURATION_NAME_1_3));
```

Comment:
Please line wrap `));`, always think about how to reduce diff on extension of collections.

Reply:
done